lib/ops/opsservice/deploy.go

```diff
@@ -117,7 +117,8 @@ func remoteDirectories(operation ops.SiteOperation, server *ProvisionedServer, m
 		server.InGravity("planet", "state"),
 		server.InGravity("site"),
 		server.InGravity("backup"),
-		server.InGravity("monitoring"),
+		server.InGravity("monitoring", "influxdb"),
```

Comment:
Don't you need to create these first?
Also, I believe this should the other way around - create these directories (i.e. list them in `directories`) and leave just the root directory (`monitoring`) here - it will be `chown`ed recursively.

Reply:
Yes, I wanted to add it into directories array. :facepalm: